pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+defects = ["pymatgen-analysis-defects>=2023.7.12", "shakenbreak>=23.6.23"]
```

Comment:
FYI: I have added a set of "extras" to the installation process so that if you do `pip install -e .[defects]`, it would install the defect dependencies. This is helpful for users.